Remotion/Web/Development.WebTesting.IntegrationTests/BrowserSessionTest.cs

```diff
@@ -34,18 +34,18 @@ public void DeleteAllCookies_DeletesAllCookies ()
     {
       var browserSession = Start();
 
-      var allCookiesBeforeCreation = ((RemoteWebDriver)browserSession.Context.Browser.Driver.Native).Manage().Cookies.AllCookies;
+      var allCookiesBeforeCreation = ((OpenQA.Selenium.WebDriver)browserSession.Context.Browser.Driver.Native).Manage().Cookies.AllCookies;
```

Comment:
It looks like we can simply switch to IWebDriver as the cast-type in this file.. Please do and verify.

Reply:
Yes, we can use IWebDriver.